date_range/views/date_range_view.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_date_range_tree" model="ir.ui.view">
+        <field name="name">date.range.tree</field>
+        <field name="model">date.range</field>
+        <field name="arch" type="xml">
+            <tree editable="bottom" string="Date range">
+                <field name="name"/>
+                <field name="type_id"/>
+                <field name="date_start"/>
+                <field name="date_end"/>
+                <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                <field name="active"/>
+            </tree>
+        </field>
+    </record>
+    <record id="view_date_range_form_view" model="ir.ui.view">
+        <field name="name">date.range.form</field>
+        <field name="model">date.range</field>
+        <field name="arch" type="xml">
+            <form string="Date Range">
+                <group col="4">
+                    <field name="name"/>
+                    <field name="type_id"/>
+                    <field name="date_start"/>
+                    <field name="date_end"/>
+                    <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                    <field name="active"/>
+                </group>
+            </form>
+        </field>
+    </record>
+    <record id="view_date_range_type_tree" model="ir.ui.view">
+        <field name="name">date.range.type.tree</field>
+        <field name="model">date.range.type</field>
+        <field name="arch" type="xml">
+            <tree editable="bottom" string="Date range type">
+                <field name="name"/>
+                <field name="allow_overlap"/>
+                <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                <field name="active"/>
+            </tree>
+        </field>
+    </record>
+    <record id="view_date_range_type_form_view" model="ir.ui.view">
+        <field name="name">date.range.type.form</field>
+        <field name="model">date.range.type</field>
+        <field name="arch" type="xml">
+            <form string="Date Range Type">
+                <group col="4">
+                    <field name="name"/>
+                    <field name="allow_overlap"/>
+                    <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                    <field name="active"/>
+                </group>
+            </form>
+        </field>
+    </record>
+    <record id="date_range_action" model="ir.actions.act_window">
+        <field name="name">Date Ranges</field>
+        <field name="type">ir.actions.act_window</field>
```

Comment:
```suggestion
```

Reply:
I see in odoo many examples where this field is not removed yet :S

Reply:
Field `type` was removed from ir_act_window in 13.0. So `<field name="type">ir.actions.act_window</field>` must be removed as well.

If Odoo didn't removed it... their bad.

Reply:
No, `type` was not removed. What was removed was `view_type`.

Reply:
Right, sorry.
Anyway `ir.actions.act_window` is the default value for `type`.
So that line can be safely removed.